### DISCUSSION TITLE
Move color mode switch to the left in navbar

### DISCUSSION THREAD

**Discussion by LuckeeDev** [facebook/docusaurus#8695] [Q&A] [answered]
In the default Docusaurus navbar configuration the color mode switch is not present, but it's rendered anyways. I would like to move this switch to the left side of the navbar, how can I do this?

**Comment by slorber** [comment, marked as answer]
This is an issue I'm aware of. 

Unfortunately, the search item and color mode switch item have been historically hardcoded.
Now we can move the search to the right: http://localhost:3000/docs/api/themes/configuration#navbar-search

We should do the same for the color mode switch but it's not done yet. Until then you can only swizzle eject Navbar/Content

**Reply by LuckeeDev** [reply]
Thanks, @slorber!
